src/node.rs

```diff
@@ -35,12 +41,21 @@ pub struct Node {
     /// Specify the network of this node.
     #[structopt(default_value = "2", short = "n", long = "network")]
     pub network: u16,
+    /// The listener IP of the node.
+    #[structopt(default_value = "0:0:0:0", long = "ip")]
+    pub ip: String,
     /// Specify the port for the node server.
-    #[structopt(long = "node")]
-    pub node: Option<u16>,
+    #[structopt(long = "listener-port")]
+    pub listener_port: Option<u16>,
+    /// Disable the RPC server.
+    #[structopt(long = "disable-rpc")]
+    pub disable_rpc: bool,
```

Comment:
This is very messy, let's take another design look.

Reply:
What is messy about it? Not spawning an RPC server in non-RPC tests makes them a lot faster, so it's a desirable feature. Alternatively I could disable that block directly using `#[cfg(not(feature = "test"))]`.